app/templates/agenda/agendaitems/agendaitem/documents.hbs

```diff
@@ -119,4 +119,28 @@
   </WebComponents::VlModal>
 {{/if}}
 
-<Documents::LinkedDocuments @agendaitemOrSubcase={{this.agendaitem}} />
\ No newline at end of file
+
+{{#if this.isOpenVerifyUploadOnApproved}}
+  <Auk::Modal @size="small">
+    <Auk::Modal::Header
+      @title={{t "documents-on-approved-agenda-title"}}
+      @onClose={{set this "isOpenVerifyUploadOnApproved" false}}
+    />
+    <Auk::Modal::Body>
+      {{#if this.currentAgenda.nextVersion}}
+        <p>{{t "documents-on-approved-agenda-with-next-version"}}</p>
+      {{/if}}
+      <p>{{t "documents-on-approved-agenda-message"}}</p>
+    </Auk::Modal::Body>
+    <Auk::Modal::Footer @onCancel={{set this "isOpenVerifyUploadOnApproved" false}}>
+      <Auk::Button
+        @skin="primary"
+        {{on "click" this.verifyUploadOnApproved}}
+      >
+        {{t "confirm"}}
+      </Auk::Button>
+    </Auk::Modal::Footer>
+  </Auk::Modal>
+{{/if}}
```

Comment:
Can we reuse the `ConfirmationModal` component that is introduced in the user-management branches? Eg. https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/templates/settings/users/index.hbs#L278-L294

Reply:
o.O new modal?
Will implement!
I can see that this modal does not except custom button skins yet (f.e. a warning style confirm button) but I guess you can just use a `:footer block` in that case.